plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -225,6 +231,71 @@ public BigQueryConfig setQueryResultsCacheEnabled(boolean queryResultsCacheEnabl
         return this;
     }
 
+    public int getRpcInitialChannelCount()
+    {
+        return rpcInitialChannelCount;
+    }
+
+    @ConfigHidden
```

Comment:
Any reason we want to hide these properties? Because it's marked as `@BetaApi`?

Reply:
Yes. I don't want to expose them yet.